### ISSUE TITLE
Error when generating docs for JSDoc

### ISSUE THREAD

**Issue by hegemonic** [jsdoc3/jsdoc#914]
Running `node jsdoc.js jsdoc.js lib/jsdoc/* -r` currently results in the following error on master:

```
/Users/jeffw/github/jsdoc3/jsdoc/master/lib/jsdoc/util/templateHelper.js:236
    return str.replace(/&/g, '&amp;')
               ^
TypeError: Object false has no method 'replace'
    at exports.htmlsafe (/Users/jeffw/github/jsdoc3/jsdoc/master/lib/jsdoc/util/templateHelper.js:236:16)
    at eval (eval at <anonymous> (/Users/jeffw/github/jsdoc3/jsdoc/master/node_modules/underscore/underscore.js:1329:20), <anonymous>:107:14)
    at Array.forEach (native)
    at eval (eval at <anonymous> (/Users/jeffw/github/jsdoc3/jsdoc/master/node_modules/underscore/underscore.js:1329:20), <anonymous>:71:16)
    at template (/Users/jeffw/github/jsdoc3/jsdoc/master/node_modules/underscore/underscore.js:1336:21)
    at exports.Template.partial (/Users/jeffw/github/jsdoc3/jsdoc/master/lib/jsdoc/template.js:60:29)
    at eval (eval at <anonymous> (/Users/jeffw/github/jsdoc3/jsdoc/master/node_modules/underscore/underscore.js:1329:20), <anonymous>:55:14)
    at template (/Users/jeffw/github/jsdoc3/jsdoc/master/node_modules/underscore/underscore.js:1336:21)
    at exports.Template.partial (/Users/jeffw/github/jsdoc3/jsdoc/master/lib/jsdoc/template.js:60:29)
    at eval (eval at <anonymous> (/Users/jeffw/github/jsdoc3/jsdoc/master/node_modules/underscore/underscore.js:1329:20), <anonymous>:203:14)
```

**Comment by hegemonic** [comment]
It appears that 028c65c037d20cfe85314b858cf9c89449306335 is at fault.

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3.
